code/Model/SiteTree.php

```diff
@@ -1914,6 +1916,7 @@ public function getCMSFields()
                         );
                     }
                 ));
+            $dependentTable->getConfig()->addComponent(new GridFieldLazyLoader());
```

Comment:
Please use Injector to instantiate classes

Reply:
None of the Gridfield components are injectable. Not sure if there's an underlying reason for this or if it's just an omission.

If we think it would be better to make them injectable, then we should probably have a separate job for this.

Reply:
Yep, that's right. I'm happy to have a separate issue for it though